pkg/controller/handle.go

```diff
@@ -0,0 +1,57 @@
+package controller
+
+import (
+	"fmt"
+	"log"
+	"regexp"
+
+	networkv1 "k8s.io/api/networking/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	kubeSystem = "kube-system"
+)
+
+var policy = &networkv1.NetworkPolicy{
+	ObjectMeta: metav1.ObjectMeta{
+		Name: "allow-all-except-metadata",
+	},
+	Spec: networkv1.NetworkPolicySpec{
+		PolicyTypes: []networkv1.PolicyType{networkv1.PolicyTypeEgress},
+		Egress: []networkv1.NetworkPolicyEgressRule{
+			{
+				To: []networkv1.NetworkPolicyPeer{
+					{
+						IPBlock: &networkv1.IPBlock{
+							CIDR:   "0.0.0.0/0",
+							Except: []string{"169.254.169.254/32"},
```

Comment:
Let's make this const and name it `cloudMetadataAddress` in order to make it more readable. A comment might be sufficient enough as well.